homeassistant/components/siren/services.yaml

```diff
@@ -0,0 +1,41 @@
+# Describes the format for available siren services
+
+turn_on:
+  description: Turn siren on.
+  target:
+    entity:
+      domain: siren
+  fields:
+    tone:
+      description: The tone to emit when turning the siren on. Must be supported by the integration.
+      example: fire
+      required: false
+      selector:
+        text:
+    volume_level:
+      description: The volume level of the noise to emit when turning the siren on. Must be supported by the integration.
+      example: 0.5
+      required: false
+      selector:
+        number:
+          min: 0
+          max: 1
+          step: 0.05
+    duration:
+      description: The duration in seconds of the noise to emit when turning the siren on. Must be supported by the integration.
+      example: 15
+      required: false
+      selector:
+        text:
```

Comment:
Should this be a number instead?

Reply:
`number` requires a max. Is there a max we should set?

Reply:
Does it require both max and min?

Reply:
Yeah: https://www.home-assistant.io/docs/blueprint/selectors/#number-selector

Reply:
I always thought that was odd. Maybe something we should change?

Reply:
Oh but the problem is that the slider needs a min and max and it's the default UI

Reply:
Ok